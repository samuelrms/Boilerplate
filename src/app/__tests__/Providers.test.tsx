import { render } from '@testing-library/react';

import Providers from '../providers';

describe('Providers', () => {
  test('renders Providers component', () => {
    const { getByTestId } = render(
      <Providers>
        <div data-testid="test-child">Test Child</div>
      </Providers>
    );

    expect(getByTestId('test-child')).toBeInTheDocument();
  });

  test('should render correctly', () => {
    const { container } = render(
      <Providers>
        <div data-testid="test-child">Test Child</div>
      </Providers>
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
