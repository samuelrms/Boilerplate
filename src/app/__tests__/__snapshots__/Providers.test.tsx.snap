// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Providers should render correctly 1`] = `
<div
  data-testid="test-child"
>
  Test Child
</div>
`;
